recipes/easybuild-framework/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "easybuild-framework" %}
+{% set version = "3.1.0" %}
+{% set md5 = "f2d06240467f06c98fb2e9243ce5d8ef" %}
+
+package:
+  name: {{ name }} 
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/fd/dd/83d8ef6a981c9e953f523328d5fba23b1be84fb7640a2c1d8b8e66b07102/{{name}}-{{version}}.tar.gz
+  md5: f2d06240467f06c98fb2e9243ce5d8ef
+
+build:
+  number: 0
+  skip: True  # [win or py3k]
+  script: |
+     python setup.py install  \
+     --single-version-externally-managed \
+     --record=record.txt 
+
+requirements:
+  build:
+    - python
+    - setuptools >=0.6
+    - pyyaml
+    - coloredlogs
+    - humanfriendly
+    - vsc-install >=0.9.19
+    - vsc-base >=2.5.4
+
+  run:
+    - python
+    - setuptools >=0.6
+    - pyyaml
+    - coloredlogs
+    - humanfriendly
+    - vsc-install >=0.9.19
+    - vsc-base >=2.5.4
```

Comment:
there are a couple of additional (optional) dependencies that may be interesting to include, e.g. `keyring` (for `eb --new-pr`), `GC3Pie` (for `eb --job`), `GitPython` (`eb --new-pr`), `pep8` (`eb --style-check`)